sdk/confidentialledger/azure-confidentialledger/azure/confidentialledger/_patch.py

```diff
@@ -0,0 +1,109 @@
+# ------------------------------------
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+# ------------------------------------
+"""Customize generated code here.
+
+Follow our quickstart for examples: https://aka.ms/azsdk/python/dpcodegen/python/customize
+"""
+import os
+from typing import Any, List, Union
+
+from azure.core.credentials import TokenCredential
+from azure.core.pipeline import policies
+
+from azure.confidentialledger._client import ConfidentialLedgerClient as GeneratedClient
+from azure.confidentialledger.certificate import ConfidentialLedgerCertificateClient
+
+__all__: List[str] = [
+    "ConfidentialLedgerCertificateCredential",
+    "ConfidentialLedgerClient",
+]  # Add all objects you want publicly available to users at this package level
+
+
+def patch_sdk():
+    """Do not remove from this file.
+
+    `patch_sdk` is a last resort escape hatch that allows you to do customizations
+    you can't accomplish using the techniques described in
+    https://aka.ms/azsdk/python/dpcodegen/python/customize
+    """
+
+
+class ConfidentialLedgerCertificateCredential:
+    """A certificate-based credential for the ConfidentialLedgerClient.
+
+    :param certificate_path: Path to the PEM certificate to use for authentication.
+    :type certificate_path: Union[bytes, str, os.PathLike]
+    """
+
+    def __init__(self, certificate_path: Union[bytes, str, os.PathLike]):
+        self._certificate_path = certificate_path
+
+    @property
+    def certificate_path(self) -> Union[bytes, str, os.PathLike]:
+        """The path to the certificate file for this credential."""
+
+        return self._certificate_path
+
+
+class ConfidentialLedgerClient(GeneratedClient):
+    """The ConfidentialLedgerClient writes and retrieves ledger entries against the Confidential
+    Ledger service.
+
+    :ivar confidential_ledger: ConfidentialLedgerOperations operations
+    :vartype confidential_ledger: azure.confidentialledger.operations.ConfidentialLedgerOperations
```

Comment:
Can we remove this ivar from the docstring?
I think this is why the operations group is still showing up in the APIview.

Reply:
Oh shoot, I missed that I had copied this from an older autogen when I patched this. Thanks for picking up on this!